### ISSUE TITLE
Creating multiple projects do not create new instances

### ISSUE THREAD

**Issue by cgsmith** [flowforge/flowforge#333]
### Current Behavior

I am running 0.2 locally for testing. I created separate projects under 3 different teams. The first project created was assigned 7880 for a port. I then created some flows within the project successfully.

I created a new project and saw it provisioned 7881 for the port. Curious, I went to create a second project within the new team and it still used port 7881. 

### Expected Behavior

I expected a new instance to be created with the new project.

### Steps To Reproduce

Install and run flowforge locally: 

```
[2022-03-04T03:38:38.904Z] INFO: FlowForge v0.2.0
[2022-03-04T03:38:38.906Z] INFO: FlowForge running with NodeJS v16.13.2
[2022-03-04T03:38:38.906Z] INFO: FlowForge Data Directory: /opt/flowforge
[2022-03-04T03:38:38.906Z] INFO: Config File: /opt/flowforge/etc/flowforge.yml
[2022-03-04T03:38:38.915Z] INFO: Database driver: sqlite
[2022-03-04T03:38:38.915Z] INFO: Database file: /opt/flowforge/var/forge.db
[2022-03-04T03:38:38.998Z] INFO: No license applied
[2022-03-04T03:38:39.046Z] INFO: Email not configured
[2022-03-04T03:38:39.066Z] INFO: Container driver: localfs
[2022-03-04T03:38:39.087Z] INFO: Server listening at http://127.0.0.1:3000
```

1. Create new team
2. Create new project
3. Add inject and debug nodes into flow 1 - deploy
4. Create new team
5. Create new project (port 7881)
6. Create new project again (same port - expected to be different port)

### Environment

- FlowForge version: 0.2.0
- Node.js version: 16.13.2
- npm version: 8.1.2
- Platform/OS: PopOS! 20.04 LTS
- Browser: Firefox

**Comment by cgsmith** [comment]
I found this interesting... I tried creating a new project under a test team... It said the port is 7881 event though it is a new project directory:

```
[2022-03-04T03:42:34.652Z] INFO: Creating userDir /opt/flowforge/var/projects/9929c8a7-a6a0-430f-97b5-6d673438f0f4
[2022-03-04T03:42:34.719Z] INFO: PID 55829, port, 7881, directory, /opt/flowforge/var/projects/9929c8a7-a6a0-430f-97b5-6d673438f0f4
```

**Comment by cgsmith** [comment]
Here is the out.log from that project. May lead to the source of the error.
[out.log](https://github.com/flowforge/flowforge/files/8182910/out.log)

**Comment by hardillb** [comment]
This feels like a race condition somewhere in the localfs driver when it assigns port numbers. I can recreate locally so will dig

**Comment by cgsmith** [comment]
I'll test out on docker as well. The EADDRINUSE made it seem like it had a collision with a port.

**Comment by hardillb** [comment]
The docker setup is totally different so without a LOT more information about what you actually did it is not going to be related to what you have described earlier.

**Comment by hardillb** [comment]
OK, I think I have a fix for this. Will test a bit more and then decide if we push a fix early or wait for 0.3.0

**Comment by hardillb** [comment]
Hi, this should be fixed now, it took a little while to publish the fix because we uncovered another problem while fixing this.

@flowforge/localfs v0.2.1 includes the fix.

Easiest way to test should be
1. Remove any projects with clashing port numbers in the Admin interface
2. Stop FlowForge
3. Assuming installed in `/opt/flowforge`, cd into `/opt/flowforge/app` and run `npm install @flowforge/localfs@0.2.1`
4. Restart FlowForge
